dev/integration_tests/flutter_gallery/lib/demo/material/buttons_demo.dart

```diff
@@ -250,7 +250,7 @@ class _ButtonsDemoState extends State<ButtonsDemo> {
               OutlinedButton.icon(
                 icon: const Icon(Icons.add, size: 18.0),
                 label: const Text('DISABLED', semanticsLabel: 'DISABLED BUTTON 6'),
-                onPressed: null,
+                onPressed: () {},
```

Comment:
I think this should be `null` so that the button stays disabled.

Reply:
Done